packages/frontend-core/src/components/grid/cells/AttachmentCell.svelte

```diff
@@ -12,7 +12,7 @@
   export let schema
   export let maximum
 
-  const { API, notifications } = getContext("grid")
+  const { API, notifications, isCloud } = getContext("grid")
```

Comment:
This will need to be `props` - we store all grid props in a top level `props` store in grid context, but memoize certain props (see the `config` store) and provide those top level for performance so that we can react to them in isolation, but we don't need to do that here.

Reply:
good catch, updated!